### DISCUSSION TITLE
Remove (no tags) tag?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [VueTorrent/VueTorrent#1400] [Q&A] [answered]
I'm loving the rebuild (I just installed it)! Thanks for doing this! One question though: Is there any way to not display the (no tags) tag when a torrent has no tags? I'm very visual, and it makes it difficult to see at a glance things that don't have tags.

**Comment by SawkeeReemo** [comment]
Oh and looks like it's also doing that for categories. Would love the option to turn that off, please.

**Comment by WDaan** [comment, marked as answer]
Delivered in 2.4.0, see #1380

**Comment by SawkeeReemo** [comment]
Awesome. I don’t seem to have access to anything beyond 2.3.0. Assuming we’re just waiting for an official release then? (Sorry, I’m not sure how GitHub works on the dev side.)

**Reply by Larsluph** [reply]
That's it. 2.4.0 just got out today

**Comment by WDaan** [comment]
Not released yet. Open PR. Will release now

**Reply by SawkeeReemo** [reply]
Thanks! Have a great weekend!
